rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -739,6 +739,13 @@ stream {
         ssl_stapling                            on;
         ssl_stapling_verify                     on;
         {{ end }}
+
+        {{ if and (not $all.DisableLua) $all.DynamicConfigurationEnabled }}
+        ssl_certificate_by_lua_block {
+            local lua_certificate = require("certificate")
```

Comment:
Let's do this in the `init` phase - see how other modules are required https://github.com/hnrytrn/ingress-nginx/blob/e90c78e617aa1c0829983a515061f69bb63830ec/rootfs/etc/nginx/template/nginx.tmpl#L56